### ISSUE TITLE
Instant App Launcher error

### ISSUE THREAD

**Issue by mjb10** [instantOS/instantOS#124]
The Instant App Launcher displays with the next message:

"
the following errors were detecter when starting rofi:

/home/$user/.config/rofi/config.rasi:1:16: The option:

configuration
{
theme: "~/.config/rofi/arc.rasi";
}
is depretated. Please replace with: @theme "~/.config/rofi/arc.rasi" after the configuration block.
"

I already tried to replace de line with that replacement, but gives me a sintax error.

**Comment by mjb10** [comment]
Almost forgot: I recently updated the rofi package. Now every boot displays the error and I cannot use mi Apps like before.

**Comment by con-f-use** [comment]
fixed in: https://github.com/instantOS/instantTHEMES/pull/3

**Comment by mjb10** [comment]
Ok. The Issue can be fixed with that fix posted on instantThemes but...

The bad thing is that the file in './config/rofi/config.rasi' always get to the previous config on every boot and show the error in the App launcher. So, on every boot, I have to keybind to the terminal and change the file. 

I saw that the 'utils/functions.sh' on InstantThemes repo was updated and I understand the fix. The problem is that I cannot find that file to change it. Maybe updating that file solves de issue? where is it?

Thanks
